server/src/test/java/org/elasticsearch/cluster/coordination/CoordinationStateTests.java

```diff
@@ -752,6 +759,11 @@ public void testVoteCollection() {
             });
     }
 
+    public void testSafety() {
+        Cluster cluster = new Cluster(randomIntBetween(1, 5));
+        cluster.runRandomly(10000);
```

Comment:
IntelliJ suggests to inline this constant, and while I'm here I suggest to inline `cluster`.

Reply:
fixed in 711447b